.eslintcache

```diff
@@ -0,0 +1 @@
{"kind":"5260","justification":"5261"},[7035,7038],{"kind":"5260","justification":"5261"},[7041,7042],{"kind":"5260","justification":"5261"},[7046,7047],{"kind":"5260","justification":"5261"},[7057,7058],{"kind":"5260","justification":"5261"},[7066,7067],{"kind":"5260","justification":"5261"},[7092,7093],{"kind":"5260","justification":"5261"},[7116,7117],{"kind":"5260","justification":"5261"},[7122,7123],{"kind":"5260","justification":"5261"},[7132,7134],{"kind":"5260","justification":"5261"},[7135,7137],{"kind":"5260","justification":"5261"},[7144,7147],{"kind":"5260","justification":"5261"},[7170,7172],{"kind":"5260","justification":"5261"},[7176,7178],{"kind":"5260","justification":"5261"},[7190,7193],{"kind":"5260","justification":"5261"},[7206,7207],{"kind":"5260","justification":"5261"},[7215,7216],{"kind":"5260","justification":"5261"},[7232,7233],{"kind":"5260","justification":"5261"},[7237,7240],{"kind":"5260","justification":"5261"},[7243,7244],{"kind":"5260","justification":"5261"},[7249,7251],{"kind":"5260","justification":"5261"},[7278,7279],{"kind":"5260","justification":"5261"},[7291,7294],{"kind":"5260","justification":"5261"},[7297,7298],{"kind":"5260","justification":"5261"},[7304,7307],{"kind":"5260","justification":"5261"},[7310,7311],{"kind":"5260","justification":"5261"},[7326,7329],{"kind":"5260","justification":"5261"},[7332,7333],{"kind":"5260","justification":"5261"},[7335,7336],{"kind":"5260","justification":"5261"},[7358,7359],{"kind":"5260","justification":"5261"},[7368,7369],{"kind":"5260","justification":"5261"},[7384,7385],{"kind":"5260","justification":"5261"},[7387,7388],{"kind":"5260","justification":"5261"},[7398,7401],{"kind":"5260","justification":"5261"},[7404,7405],{"kind":"5260","justification":"5261"},[7424,7425],{"kind":"5260","justification":"5261"},[7456,7457],{"kind":"5260","justification":"5261"},[7459,7460],{"kind":"5260","justification":"5261"},[7482,7483],{"kind":"5260","justification":"5261"},[7485,7486],{"kind":"5260","justification":"5261"},[7527,7528],{"kind":"5260","justification":"5261"},[7555,7558],{"kind":"5260","justification":"5261"},[7560,7561],{"kind":"5260","justification":"5261"},[7565,7566],{"kind":"5260","justification":"5261"},[8125,8128],{"kind":"5260","justification":"5261"},[8130,8131],{"kind":"5260","justification":"5261"},[8177,8178],{"kind":"5260","justification":"5261"},[8612,8615],{"kind":"5260","justification":"5261"},[8617,8618],{"kind":"5260","justification":"5261"},[8751,8752],{"kind":"5260","justification":"5261"},[8753,8754],{"kind":"5260","justification":"5261"},[8757,8758],{"kind":"5260","justification":"5261"},[8759,8760],{"kind":"5260","justification":"5261"},[8767,8770],{"kind":"5260","justification":"5261"},[8772,8773],{"kind":"5260","justification":"5261"},[8801,8804],{"kind":"5260","justification":"5261"},[8806,8807],{"kind":"5260","justification":"5261"},[8810,8811],{"kind":"5260","justification":"5261"},[8884,8885],{"kind":"5260","justification":"5261"},[89
```

Comment:
This file needs to removed from git index in order for `.gitignore` to be effective.

Reply:
Not sure how that snuck in there